scripts/mutect2_wdl/mutect2_nio.wdl

```diff
@@ -168,16 +168,56 @@ workflow Mutect2 {
     # Small is for metrics/other vcfs
     Float large_input_to_output_multiplier = 2.25
     Float small_input_to_output_multiplier = 2.0
+    Float cram_to_bam_multiplier = 6.0
 
     # logic about output file names -- these are the names *without* .vcf extensions
-    String output_basename = basename(tumor_bam, ".bam")
+    String output_basename = basename(basename(tumor_reads, ".bam"),".cram")  #hacky way to strip either .bam or .cram
     String unfiltered_name = output_basename + "-unfiltered"
     String filtered_name = output_basename + "-filtered"
     String funcotated_name = output_basename + "-funcotated"
 
-    String output_vcf_name = basename(tumor_bam, ".bam") + ".vcf"
+    String output_vcf_name = basename(tumor_reads, ".bam") + ".vcf"
```

Comment:
This line should probably say `String output_vcf_name = output_basename + ".vcf"`

Reply:
true